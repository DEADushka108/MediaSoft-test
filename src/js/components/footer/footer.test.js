import React from 'react';
import renderer from 'react-test-renderer';
import {MemoryRouter} from 'react-router-dom';
import Footer from './footer';

it(`Should render Footer component correctly`, () => {
  const tree = renderer
    .create(
        <MemoryRouter>
          <Footer/>
        </MemoryRouter>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
