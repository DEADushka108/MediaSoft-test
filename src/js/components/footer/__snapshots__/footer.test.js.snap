// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Footer component correctly 1`] = `
<footer
  className="page-footer"
>
  <div
    className="page-footer__wrapper"
  >
    <a
      className="page-footer__logo"
      href="/"
      onClick={[Function]}
    >
      Starshipsshop
    </a>
    <div
      className="page-footer__copyright"
    >
      <p
        className="page-footer__text"
      >
        Â© 2020 Stars ships shop Ltd.
      </p>
    </div>
  </div>
</footer>
`;
